.golangci.yml

```diff
@@ -0,0 +1,19 @@
+linters-settings:
+    golint:
+        min-confidence: 0.9
+
+    gocyclo:
+        min-complexity: 15
+
+linters:
+    enable:
+        - misspell
+        - gofmt
+        - golint
+        - goimports
+    disable:
+        - errcheck
```

Comment:
We might want to tweak the linters a bit given that it's brand new code. Errcheck for example is quite helpful (at least in theory)